packages/wayland/compositor/sway/config/config

```diff
@@ -0,0 +1,243 @@
+# Default config for sway
```

Comment:
Did you want to put a copyright on this file?

Reply:
It's mostly the default sway config file with some additions to run bemenu, foot & it adjust colors/time format. No clue if it's worth to add copyright